### PR TITLE
Add Zendesk api wrapper functions

### PR DESCRIPTION
**Pull request by aubin-tchoi**
## Description

- ~~Add wrapper functions around Zendesk API (lots of line but lowkey boilerplate).~~
- Add the package `node-zendesk`.
- Add a few types on top of `node-zendesk`.
- Update the OAuth scopes accordingly (we are gonna need more than the two scopes already defined).

## Deploy Plan

- Deploy core
- Deploy front
- Deploy connectors

### CODE REVIEW

**Comment by PopDaph** [comment]
@aubin-tchoi last update of the repo was 2 months ago: https://github.com/blakmatrix/node-zendesk
It seems that it is still supported and alive no?

File: connectors/src/connectors/zendesk/lib/zendesk_api.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { createClient } from "node-zendesk";
+
+export function createZendeskClient({
+  token,
+  subdomain = "d3v-dust",
```
Comment (PopDaph) [inline connectors/src/connectors/zendesk/lib/zendesk_api.ts:5]: What's d3v-dust?

**Review by PopDaph — APPROVED** [review]
LGTM

File: connectors/src/connectors/zendesk/lib/zendesk_api.ts (hunk 1 above)
Comment (aubin-tchoi) [reply connectors/src/connectors/zendesk/lib/zendesk_api.ts:5]: it's the subdomain we use for tests, in a future work we'll fix the authentication flow to retrieve the client's subdomain and use it here

**Pull request merged**